.github/workflows/ruby_ci.yml

```
@@ -0,0 +1,19 @@
+name: CI
+
+on: pull_request
+
+concurrency:
+  group: ${{ github.ref_name }}-${{ github.workflow }}
+  cancel-in-progress: true
+
+jobs:
+  rubocop:
+    name: Rubocop
+    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
```

> I raised it before, but should we start assuming Rubocop as an executable exists and always use that?

For now I'd go with specifying the exact command.

```suggestion
    uses: theforeman/actions/.github/workflows/rubocop.yml@v0
    with:
      command: bundle exec rubocop --parallel --format github
```